src/alliteratedActors.js

```diff
@@ -22,7 +22,7 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  return names.find(item => item.split(' ').every(i => item[0] === i[0]));
```

Comment:
`item`, `i` bad variable names.
Better `nameParts`, `firstLetter` or something like that.
Variable name must describe it.